src/main/java/org/spongepowered/api/data/DataTransactionResult.java

```diff
@@ -206,6 +207,24 @@ public static DataTransactionResult errorResult(final ImmutableValue<?> value) {
         return builder().result(Type.ERROR).reject(value).build();
     }
 
+    /**
+     * Represents the possible states of a value in this data transaction result
+     */
+    public enum DataCategory {
+        /**
+         * Represents values that were successfully offered to the {@link DataHolder}
+         */
+        SUCCESSFUL,
```

Comment:
How about `NEW`?

Reply:
This corresponds to `#getSuccessfulData`, so it should be called `SUCCESSFUL`

Reply:
Does it also refer to the succeed data after replacement?